packages/tools/viewer-configurator/src/components/babylonViewer/viewer.tsx

```diff
@@ -6,7 +6,7 @@ import { Logger } from "core/Misc/logger";
 import { ConfigureCustomViewerElement } from "viewer/viewerElement";
 import "viewer";
 
-interface HTML3DElementAttributes extends React.DetailedHTMLProps<React.HTMLAttributes<HTMLElement>, HTMLElement> {
+interface IHTML3DElementAttributes extends React.DetailedHTMLProps<React.HTMLAttributes<HTMLElement>, HTMLElement> {
```

Comment:
Hmm, this gets weird where we have to decide whether it makes more sense to align with Babylon naming conventions or React naming conventions.

Reply:
Can be discussed, if you want to raise a concern here